.github/workflows/discord-posts.yml

```diff
@@ -9,14 +9,37 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.pull_request.merged == true &&
       github.event.pull_request.base.ref == 'master'
+    env:
+      CHECK_NAME: Internal Tests
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
+      - name: Set up GitHub CLI
+        run: |
+          curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo tee /usr/share/keyrings/githubcli-archive-keyring.gpg > /dev/null
+          sudo apt-get install -y apt-transport-https
+          echo "deb [arch=amd64 signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list
+          sudo apt-get update
+          sudo apt-get install gh
```

Comment:
I believe the `spacetimedb-runner` already includes the GH CLI, but I don't know the other implications of using that runner.